src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {
```

Comment:
```suggestion
public class Ball {
```
Ball should not extend Lottery. Can we say "Ball is a Lottery"? Like "Cat is an Animal" or "BMW is a Car".